maqs-webservices/src/test/java/com/cognizantsoftvision/maqs/webservices/WebServiceConfigUnitTest.java

```diff
@@ -5,15 +5,14 @@
 
 package com.cognizantsoftvision.maqs.webservices;
 
-import com.cognizantsoftvision.maqs.base.BaseGenericTest;
 import com.cognizantsoftvision.maqs.utilities.helper.TestCategories;
 import org.testng.Assert;
 import org.testng.annotations.Test;
 
 /**
  * The Web Service Config unit test class.
  */
-public class WebServiceConfigUnitTest extends BaseGenericTest {
+public class WebServiceConfigUnitTest extends BaseWebServiceTest {
```

Comment:
Why are we extending base service if it is not required for the tests?

Reply:
changed back to baseGenericTest